Finished the tutorial

isibley765 opened the pull request:
Edit: two files up top (new_id.js and db.js) and one at the very bottom (server.js) that are the main edits. The rest was deleting the mongodb file. I'll make that a separate commit next time, to make it more clear.

Got everything to work through mysql instead of mongoose.

Noticed I'd only deleted mongoose earlier and not mongodb, so I fixed that, just to make sure everything was now mysql-dependent only. It was.

The tools and connections pathways are files that server.js pulls in to run its functionality without crowding up the code. 

The new_id.js in tools has code commented out, that I was unable to get to work; it wanted me to make a connection locally to the database, but the server then crashed because there were two "Handshakes"--connections to the file from what I understood. 
They could be updated later. The extra functions only checked if the id's weren't already assigned; my method had a minute enough chance of matching id's already that I figured, for the time being, applying them could be put aside for now.

isibley765 committed:
Finished the tutorial

Got everything to work through mysql instead of mongoose.
Noticed I'd only deleted mongoose earlier and not mongodb, so I fixed
that.
The tools and connections pathways are files that server.js pulls in to
run its functionality without crowding up the code.

bsalter commented on app/tools/new_id.js:
```diff
@@ -0,0 +1,52 @@
+	
+	
+	create: function() {		//creates an ID tag in hexidecimal form
+	var hexNum;
```
Indent here.

bsalter commented:
You should use .gitignore to ignore your node_modules dir (and any other modules dir), so those file changes don't end up in the repo.

isibley765 committed:
Includes Ben's comments

No difference from the last pull request other than the addition of
Ben's comments. This pull request will be merged in place of the last
one.
